docs/transformations/README.md

```diff
@@ -590,3 +590,31 @@ df_transformed.show()
 +-------+---+---------+
 +-------+---+---------+
 ```
+
+## AddTimeStampTransformer
+
+This transformer adds a timestamp for when the row was ingested. It does so by creating a new column in which it takes the current timestamp.
```

Comment:
I disagree a bit with the formulation. I wouldn't say it necessarily has something to do with when the row was ingested. "The transformer adds a column with the current timestamp. The name of the column can be specified with the col_name argument."